Remotion/Development/Web/UnitTesting/UI/Controls/Rendering/HtmlHelperBase.cs

```diff
@@ -192,10 +194,11 @@ public void AssertAttribute (XmlNode node, string attributeName, string attribut
     /// </summary>
     public void AssertAttribute (XmlNode node, string attributeName, string attributeValue, AttributeValueCompareMode mode)
     {
-      XmlAttribute attribute = node.Attributes[attributeName];
+      // TODO: Assert node.Attributes not null
+      XmlAttribute? attribute = node.Attributes![attributeName];
       AssertNotNull (attribute, "Attribute {0}.{1} does not exist.", node.Name, attributeName);
 
-      if (attributeValue != null)
+      if (attributeValue != null) // TODO: Is this needed?
```

Comment:
Don't understand the question

Reply:
The parameter attributeValue is non-nullable, imo this should be asserted and the if dropped.

Reply:
Ah, I.C. okay. We should also consider argument checks since this code is used from unit tests which don't have nullablitiy enabled.